### ISSUE TITLE
Doesn't work with Angular 4 + webpack

### ISSUE THREAD

**Issue by dvdbrk** [lfabreges/nativescript-localize#8]
Hi,

So I have just updated NS to v3, and Angular to v4. Running `npm run start-android-bundle` I get the following error. I have no idea what is going on, am I doing something wrong?

> ERROR in Unexpected value 'NativeScriptLocalizeModule in C:/Users/dvd/App name/node_modules/nativescript-localize/sources/localize.d.ts' imported by the module 'SharedModule in C:/Users/dvd/App name/app/src/shared/shared.module.ts'. Please add a @NgModule annotation.

**Comment by dvdbrk** [comment]
Tried something else, different versions of other dependencies etc. and now came up with this issue:

> Executing before-prepare hook from C:\Users\dvd\App name\hooks\before-prepare\nativescript-localize.js
Cannot read property 'projectDir' of undefined
child process exited with code 127

I am using v1.9.5 with all other @angular versions set to 4.0.2.

Apologies for taking your time, but please let me know if you know what is happening on top of your head, or if I can send you some more details? Thank you

**Comment by lfabreges** [comment]
Hi, thanks for the report, could you try the 2.0.0 version and tell me if it works as expected ?

**Comment by dvdbrk** [comment]
Yes, this works now using `tns run android`. Thank you!

However, it fails when using webpack `npm run start-android-bundle --aot` with:

```
ERROR in Unexpected value 'NativeScriptLocalizeModule in C:/Users/dvdbrk/App name/node_modules/nativescript-localize/index.d.ts' imported by the module 'AppModule in C:/Users/dvdbrk/App name/app/app.module.ts'.
Please add a @NgModule annotation.

ERROR in ./main.aot.ts
Module not found: Error: Can't resolve './app.module.ngfactory' in 'C:/Users/dvdbrk/App name/app'
 @ ./main.aot.ts 4:29-62
```

I'd like to help, but webpack and all this is way above me. Any ideas?

**Comment by lfabreges** [comment]
Good, I've just published 2.0.1 to correctly generate angular metadata. You'll have to change your import from `nativescript-localize` to `nativescript-localize/angular` to make it work. Take a look at the angular demo if needed.

**Comment by dvdbrk** [comment]
I can confirm that 2.0.1 works with Webpack --aot, NS 3, Angular 4, Android. Thank you for an awesome plugin.
